src/python/WMCore/Services/ReqMgrAux/ReqMgrAux.py

```diff
@@ -410,8 +410,12 @@ def listDiskUsageOverThreshold(config, updateDB):
 
     if updateDB and not t0Flag:
         agentDrainMode = bool(len(overThresholdDisks))
-        if agentConfig and (agentDrainMode != agentConfig["AgentDrainMode"]):
-            reqMgrAux.updateWMAgentConfig(config.Agent.hostName, {"AgentDrainMode": agentDrainMode},
+        if agentConfig and (agentDrainMode != agentConfig.get("AgentDrainMode")):
+            if agentConfig.get("UserDrainMode") is True:
+                    logging.info("Agent is already in UserDrainMode: True mode. No need to update.")
```

Comment:
It looks like you got wrong indentation in this line. As python is picky with those, can you please correct it?
The jenkins report (at the bottom, pycodestyle) is giving you hints of where those are.